### PR TITLE
Remove `snow.DefaultContextTest`

### PR DESCRIPTION
**Pull request by dhrubabasu**
## Why this should be merged

We should be using `snowtest.Context()` in most places.

## How this works

Replace

## How this was tested

CI

### CODE REVIEW

**Review by abi87 — COMMENTED** [review]
change looks fine, but compilation is broken. WIll approve once CI gets to green

**Review by abi87 — APPROVED** [review]

File: snow/context.go [hunk 1]
```diff
@@ -96,22 +96,3 @@ type ConsensusContext struct {
 	// True iff this chain is currently state-syncing
 	StateSyncing utils.Atomic[bool]
 }
-
-func DefaultContextTest() *Context {
```
Comment (StephenButtolph) [inline snow/context.go:100]: ❤️

File: snow/snowtest/snowtest.go [hunk 2]
```diff
@@ -35,9 +38,28 @@ func (noOpAcceptor) Accept(*snow.ConsensusContext, ids.ID, []byte) error {
...
+		SubnetID:     ids.Empty,
+		ChainID:      ids.Empty,
+		NodeID:       ids.EmptyNodeID,
+		PublicKey:    pk,
+		Log:          logging.NoLog{},
+		BCLookup:     ids.NewAliaser(),
+		Metrics:      metrics.NewOptionalGatherer(),
+		ChainDataDir: "",
+	}
+}
```
Comment (StephenButtolph) [inline snow/snowtest/snowtest.go:58]: I know this is what we had before... but I still hate this... Are there places that we currently use `EmptyContext` that we could just be using `Context`?

**Review by StephenButtolph — APPROVED** [review]

File: snow/snowtest/snowtest.go (hunk 2 above)
Comment (dhrubabasu) [reply snow/snowtest/snowtest.go]: Removed it altogether

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**